ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.19.yaml

```diff
@@ -37,15 +37,25 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: installer-rehearse-ibmcloud-key
```

Comment:
As usual, we do not need to add so many rehearse jobs, if you like to run them in parallel, pls remove them after your testing.